src/sys/socket/mod.rs

```diff
@@ -121,6 +121,24 @@ pub enum SockType {
     #[cfg(not(any(target_os = "haiku")))]
     Rdm = libc::SOCK_RDM,
 }
+// The TryFrom impl could've been derived using libc_enum!.  But for
+// backwards-compatibility with Nix-0.25.0 we manually implement it, so as to
```

Comment:
@asomers I'm interpreting this as driving a patch, is that the intent?

Reply:
You mean so we can build patch releases?  Yes, I think that's worthwhile.  But even for new releases too I think it's annoying to force people to change SockType::Stream to SockType::SOCK_STREAM without great reason.